main.md

```diff
@@ -1089,8 +1086,8 @@ Both mechanisms MUST be supported by an authorization server that supports PAR a
 # Security Considerations {#Security}
 
 In DPoP, the prevention of token replay at a different endpoint (see
-(#objective)) is achieved through the
-binding of the DPoP proof to a certain URI and HTTP method plus the optional server-provided nonce. DPoP, however,
+(#objective)) is achieved through authentication of the server per [@!RFC6125] and
```

Comment:
I'm not sure if I'm missing something but I don't think I understand the first part this sentence. RFC6125 I think just essentially means "check the TLS certificate of the server matches the hostname", and that doesn't really prevent replay? (It might prevent a token from being leaked in the first place, but that seems slightly different to me, and it seems to do nothing to help if the attacker has obtained a DPoP proof from, say, a log file.)

Reply:
What I'm trying to convey there is that authenticating the server is part of what makes binding the proof to the URI useful in preventing replay by an attacker-in-the-middle pretending to be the real server.  That was the thinking behind that anyway. Also that that sentience start with talking about preventing "replay at a different endpoint" so the nonce part didn't make much sense to me in that context. And Nat mentioned mentioning server authentication in his WGLC review and this seemed like a nice spot to stick it in.

Reply:
That's fair. Going to proceed with it for now.